src/mpyl/reporting/formatting/markdown.py

```diff
@@ -40,36 +41,36 @@ def __add_link_if_service(name: str, output: Output) -> str:
     return name
 
 
-def wrap_project_name(proj: Project, result: list[StepResult]):
-    project_name = proj.name
+def wrap_project_name(project_execution: ProjectExecution, result: list[StepResult]):
+    project_name = project_execution.name
     encapsulation = "_"
     found_result = next((r for r in result if r.project.name == project_name), None)
     if found_result:
         project_name = __add_link_if_service(project_name, found_result.output)
         encapsulation = "*" if found_result.output.success else "~~"
 
-    return f"{encapsulation}{project_name}{encapsulation}"
+    return f"{encapsulation}{project_name}{' (cached)' if project_execution.cached else ''}{encapsulation}"
 
 
-def __to_oneliner(result: list[StepResult], plan: set[Project]) -> str:
+def __to_oneliner(result: list[StepResult], plan: set[ProjectExecution]) -> str:
     project_names: list[str] = []
     if plan:
-        sorted_plan = sorted(plan, key=operator.attrgetter("name"))
-        for proj in sorted_plan:
-            project_names.append(wrap_project_name(proj, result))
+        sorted_plans = sorted(plan, key=operator.attrgetter("name"))
+        for project_execution in sorted_plans:
+            project_names.append(wrap_project_name(project_execution, result))
     else:
         project_names = list(map(lambda r: f"_{r.project.name}_", result))
 
-    return f'{", ".join(project_names)}'
+    return f'&nbsp;  {", ".join(project_names)}'
```

Comment:
Why do we need this HTML character? Can't we just use normal space?

Reply:
If I add normal spaces it just alligns it to the left for some reason

Reply:
Can you try with `\t`?

Reply:
Same 😞